python/lsst/ip/isr/isrTask.py

```diff
@@ -2127,6 +2133,42 @@ def maskEdges(self, exposure, numEdgePixels=0, maskPlane="SUSPECT"):
                 maskBitMask
             )
 
+    def maskGrowTophat(self, exposure, boxRadius=0, boxThreshold=0.25, maskPlane="BAD", maskLabel="BAD"):
+        """Mask regions around large masked regions.
+
+        Parameters
+        ----------
+        exposure: `lsst.afw.image.Exposure`
+            Exposure to process.
+        boxRadius : `int`, optional
+            Half-size of the box to use to grow masks.
```

Comment:
How does this manifest in terms of # of pixels the original mask is "grown" by?

Reply:
In a somewhat complicated way that is probably unintuitive.  Because of this, I've taken the advice from the ticket comments, and switched to simply allowing a list of mask plane names to be grown and assigned to a new value.  This duplicates code that was used for saturation mask growing, so I've split it out.  This is faster, and provides a better expectation for the output.